### PR TITLE
fix(v7): addresses designer feedback

### PR DESCRIPTION
**Pull request by chasestarr**

### CODE REVIEW

File: src/button/constants.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ export const SHAPE = {
 };
 
 export const SIZE = {
+  medium: 'default',
```
Comment (nadiia) [inline src/button/constants.js]: We shouldn't introduce a new value that just refrences the `default`. Also, it's ok to have different naming from the designers' ones. Considering they change it pretty often, introduce new or deprecate older ones, we won't be able to follow it without breaking changes.

File: src/button/constants.js [hunk 2]
```diff
@@ -20,7 +20,9 @@ export const SHAPE = {
...
   default: 'default',
   compact: 'compact',
+  small: 'compact',
```
Comment (nadiia) [inline src/button/constants.js]: Same here. Saying that I'm inclined toward not introducing this change and stick to the current naming.

File: src/button/constants.js (hunk 1 above)
Comment (gergelyke) [reply src/button/constants.js]: I think ideally we should use the same naming to reduce friction

Comment (chasestarr) [reply src/button/constants.js]: it doesn't need to be included right now. so i'm going to remove so that this can land without deliberation

Comment (chasestarr) [reply src/button/constants.js]:
```suggestion
```

File: src/button/constants.js (hunk 2 above)
Comment (chasestarr) [reply src/button/constants.js]:
```suggestion
```

**Pull request merged**